daemon/graphdriver/vfs/copy_linux.go

```diff
@@ -0,0 +1,13 @@
+// +build linux
+
+package vfs
+
+import "github.com/docker/docker/daemon/graphdriver/copy"
+
+var (
+	CopyDir = dirCopy
```

Comment:
Can we just define `dirCopy()` as a function in both `copy_linux` and `copy_unsupported` instead of setting this var?

Reply:
This var gets changed in by  layer/layer_test.go -- why it changes it, I'm not sure. I would feel comfortable changing it, if I knew why it existed.

Reply:
fyi, this is because the `chrootarchive` archiver performs a re-exec which does some shadow magic and the unit test is not prepared for that.